Foodhut/src/sections/introduction.html

```diff
@@ -34,9 +32,9 @@ <h1 class="header-introduction">
       </include>
       <div class="watch-video-container">
         <button class="play-button" type="button">
-          <img src="./assets/images/play-button.svg" alt="play button"/>
+          <div class="play-icon"></div>
         </button>
```

Comment:
Please check a11y for this button, since this button has no content
https://accessibleweb.dev/buttons

Reply:
I fixed it, thanks for your contribution

Reply:
@CBinh710 I think solution is not matched with Hung's reference. Pls read carefully